driver_go18_test.go

```diff
@@ -247,12 +293,33 @@ func TestContextCancelExec(t *testing.T) {
 		dbt.mustExec("CREATE TABLE test (v INTEGER)")
 		ctx, cancel := context.WithCancel(context.Background())
 
-		// Delay execution for just a bit until db.ExecContext has begun.
+		var longProcess *dbProcess
+		longQuery := "INSERT INTO test VALUES (SLEEP(1))"
```

Comment:
You can use here a bigger timeout, to avoid race conditions and random fails in test. for example 15 sec.

Reply:
ok